pkg/gui/gui.go

```diff
@@ -68,7 +67,7 @@ func NewGui(log *logrus.Logger, gitCommand *commands.GitCommand, oSCommand *comm
 		ConflictTop:   true,
 		Conflicts:     make([]commands.Conflict, 0),
 		EditHistory:   stack.New(),
-		Platform:      getPlatform(),
+		Platform:      *oSCommand.Platform,
```

Comment:
Shouldn't it be OSCommand?

Reply:
OSCommand is the name of the struct type, but oSCommand is the name of the object. It looks a bit weird because of the OS acronym but I believe this is correct convention